src/rebloom.c

```diff
@@ -419,14 +414,14 @@ static int BFLoadChunk_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **arg
     if (SBChain_LoadEncodedChunk(sb, iter, buf, bufLen, &errMsg) != 0) {
         return RedisModule_ReplyWithError(ctx, errMsg);
     } else {
+        RedisModule_ReplicateVerbatim(ctx); // Should be replicated?
```

Comment:
I don't know the answer to this comment! Additionally, we can return with an OK status at 405 - does replication need to occur there?

Reply:
@gkorland @itamarhaber 
for loading chunks, should we replicate? Do we assume the load is done for the whole cluster?